.vercel/project.json

```diff
@@ -0,0 +1 @@
+{"projectId":"Qmf3ydjvNQWmSBkPurHi2GGgMbn7oQtwgZGSHJjcp4XpSa","orgId":"JAQ78UA8gBjnVXgKVRupyDO5"}
```

Comment:
Should these IDs be exposed on GitHub? If not could you look into substituting them with environment variables instead?

Reply:
My bad, forgot to add gitignore entry for vercel , i'll create a new project to invalidate the exposed IDs

Reply:
@chuabingquan removed in e929d0b and added to gitignore in e2d47d5

Reply:
Looks good to me.